drud/ddev
# Support .mysql files for import along with related files, fixes #812, fixes #516

## The Problem/Issue/Bug:

OP #812 and #516: People who use backup_migrate like to use .mysql files and the related items. This PR just allows us to import those and their variants as if they were .sql files.

## Manual Testing Instructions:

Download a mysql file, maybe from a backup_migrate instance. Use ddev  import-db on it, verify. 

## Automated Testing Overview:

This adds all variants to TestDevImportDB()

## Related Issue Link(s):

#812, #516

## Release/Deployment notes:
